vllm_ascend/quantization/utils.py

```diff
@@ -60,11 +55,17 @@ def get_linear_quant_type(quant_description: Dict[str, Any], prefix: str,
     return quant_type
 
 
+quant_logger_initialized = False
+
+
 def get_quant_method(quant_description: Dict[str, Any],
                      prefix: str,
                      layer_type: str,
                      packed_modules_mapping: Optional[Dict[str, Any]] = None):
-    apply_quantization_patch(quant_description)
+    global quant_logger_initialized
```

Comment:
use `logger.log_only_once`  function from vllm instead (I'm not sure the func name, but it supports in vllm, please take a look)